Index: main_one.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>print(\"Hello World!\")
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main_one.py b/main_one.py
--- a/main_one.py	(revision 814d1d456dd1e88659b7a25d940e39120a3a6f90)
+++ b/main_one.py	(date 1705517217154)
@@ -1,1 +1,1 @@
-print("Hello World!")
\ No newline at end of file
+print("Hello World!!!")
\ No newline at end of file
